src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs

```diff
@@ -56,9 +59,30 @@ public class PatientDiscovery
                     .Map(async patient =>
                     {
                         await discoveryRequestRepository.Add(new Model.DiscoveryRequest(request.TransactionId,
-                            request.Patient.Id, patient.Identifier));
-                        return (new DiscoveryRepresentation(patient.ToPatientEnquiryRepresentation(
-                                GetUnlinkedCareContexts(linkedCareContexts, patient))),
+                            request.Patient.Id, patient.Identifier,
+                            patient.MaskedIdentifier));
+                        var careContextRepresentations = GetUnlinkedCareContexts(linkedCareContexts, patient).ToList();
+                        patient.Identifier = patient.MaskedIdentifier;
+                        var enquiryRepresentation = patient.ToPatientEnquiryRepresentation(
+                            careContextRepresentations);
+                        if (!careContextRepresentations.Any())

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)